ArjunaJTA/jta/pom.xml

```diff
@@ -237,6 +228,18 @@
         </plugins>
       </build>
     </profile>
+    <!-- For further info about this profile, refer to https://github.com/arquillian/arquillian-extension-byteman -->
+    <profile>
+      <id>use-toolsjar-jigsaw</id>
+      <activation>
+        <jdk>[1.9,)</jdk>
```

Comment:
Java 9 is not an LTS (long term support) release and I don't think that we want to encourage its use.